frontend/src/pages/CreateTopic.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import { ITopicCreation } from "../models/ITopic";
+import { backendAPI } from "../api/api";
+import {
+     Button,
+     Center,
+     Heading,
+     Input,
+     Text,
+     useToast,
+} from "@chakra-ui/react";
+
+function CreateTopic() {
+     const [formData, setFormData] = useState<{
+          title: string;
+          description: string;
+     }>({
+          title: "",
+          description: "",
+     });
+     const toast = useToast();
+
+     function handleCreateTopic() {
+          if (
+               formData.title.trim().length === 0 ||
+               formData.description.trim().length === 0
```

Comment:
In JS you can check if an array is empty by using (!stringVariable.length). 
Here is an example : 
var emptyString=""
if (!emptyString.length)  console.log("Oh no it's empty")  //Oh no it's empty
So your whole condition can be 
if ( !(formData.title.trim().length && formData.description.trim().length) ){
   return;
 }

Reply:
thanks I will fix that !